data/mihoyo

```diff
@@ -0,0 +1,5 @@
+# 米忽悠
```

Comment:
According to https://zh.wikipedia.org/wiki/%E7%B1%B3%E5%93%88%E6%B8%B8 ,
it should be
```
#米哈游
```

Reply:
Resolved.